integration-cli/docker_cli_start_test.go

```diff
@@ -65,9 +66,8 @@ func (s *DockerSuite) TestStartAttachSilent(c *check.C) {
 	dockerCmd(c, "wait", name)
 
 	startOut, _ := dockerCmd(c, "start", "-a", name)
-	if expected := "test\n"; startOut != expected {
-		c.Fatalf("start -a produced unexpected output: expected %q, got %q", expected, startOut)
-	}
+	// start -a produced unexpected output
+	c.Assert(startOut, checker.Equals, "test\n")
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?